modules/store/src/store_module.ts

```diff
@@ -114,6 +114,14 @@ export interface RootStoreConfig<T, V extends Action = Action>
   runtimeChecks?: Partial<RuntimeChecks>;
 }
 
+/**
+ * A pair of reducer and name for the feature.
```

Comment:
To Be Fixed

Reply:
What do you mean with the comments `Shouldn't it use the word "slice" here?` and this one?
I think it's fine, perhaps the following would read a bit easier.

```suggestion
 * An object with the name and the reducer for the feature.
```

Reply:
My concern is that possibly "slice" is a word that is only meaningful in the RTK context. If not, I can feel the `FeatureSlice` type is a good name. But then, maybe I should leave a comment to address the question: "What is the Slice?".

Reply:
It doesn't bother me to be honest.
I think some sections in our docs (or in articles) are also using the term slice.
If it isn't clear, we can always add it in the future.